src/sentry/event_manager.py

```diff
@@ -23,7 +23,6 @@
     MAX_SECS_IN_FUTURE,
     MAX_SECS_IN_PAST,
 )
-from sentry.message_filters import should_filter_event
```

Comment:
These functions are dead code and can be removed (when removing them, please check what else they reference that becomes dead):
- `parse_client_as_sdk`
- `add_meta_errors`
- `process_csp_report`: This should also get rid of most of the python implementation for security reports, please check that. Also removes imports to `APIForbidden` and `APIError`, potentially making them dead code.

are not actionable now but rather for follow-ups:
- `validate_and_set_timestamp` is only used for the minidump. Once we start opening the Minidump in Relay, we can set the timestamp there and then get rid of this function. CC @flub: This could be a first small change to ship ahead of minidump scrubbing.

Reply:
APIForbidden is referenced from schemas, the rest is true